src/platform-integration/windows/extern_win.md

```diff
@@ -0,0 +1,17 @@
+---
+title: External windows in Flutter Windows apps
+description: Special considerations for adding external windows to Flutter apps
+---
+
+## Windows lifecycle
+
+When adding a non-Flutter window to a Flutter Windows app, it will not be part
+of the logic for application lifecycle state updates by default. For example,
+this means that when the external window is shown or hidden, the app lifecycle
+state will not appropriately update to inactive or hidden. In order to add the
+external window to this application logic, the window's `WndProc` procedure
+must invoke `FlutterEngine::ProcessExternalWindowMessage`.

```

Comment:
Could we mention the Flutter framework APIs that would be incorrect if you don't follow this? Perhaps [`WidgetsBindingObserver.didChangeAppLifecycleState`](https://api.flutter.dev/flutter/widgets/WidgetsBindingObserver/didChangeAppLifecycleState.html)

I'm thinking this will help folks find this page by Googling `Windows broken didChangeAppLifecycleState`.